src/AppBundle/Resources/data/posts/2016-02-10.yml

```diff
@@ -0,0 +1,41 @@
+name: "first-1-3-release-candidate"
+parent: "/news"
+title: "First Symfony CMF 1.3 Release Candidate Available"
+label: false
+format: markdown
+template: cms/news_detail.hml.twig
+publish_start_date: "2016-02-10"
+body: |
+    The release of Symfony CMF 1.3 is coming closer with the availability of
+    release candidates for all packages. Stable releases can be expected in the
+    coming weeks.
+
+    After the 1.2 release, the activity of the Symfony CMF project has dropped,
+    resulting in no new release in 2014. With this new release, the CMF is back
+    in running with full Symfony 2.8 and PHP 7 support.
+
+    ## Important new features
+    
+    The biggest features in this release are:
+
+     * Symfony 3 support for the Routing and Testing component;
+     * The CmfMenuBundle added KnpMenu 2 support and dropped support for KnpMenu 1;
+     * The CmfSeoBundle added support for generating sitemaps;
+     * Improved admin experience for most Sonata Admin classes;
+     * A ``symfony-cmf/slugifier-api`` package was introduced, containing a
+       generic sluggifier interface to be used when a sluggifier is needed;
+     * Integration of CKeditor in the CmfSimpleCmsBundle and CmfContentBundle,
+       providing the admin with better tooling to edit page contents.
+
+    You can read about all new features [master version](http://symfony.com/doc/master/cmf)
+    of the documentation.
+
+    Please update your project and to test the release candidates. This will
+    help making the 1.3 release more stable.
+
+    ## The future
+
+    Symfony CMF 1.3 will be the last release in the 1.x serie. After this
+    release, work will start on Symfony CMF 2.0. This version targets Symfony
+    2.8 and 3.0+. To provide a cleaner upgrade path, version 1.3 will trigger
+    deprecation notices when you use deprecated features.
```

Comment:
Should we also add that we will deprecate certain bundles .. or are we not decided on that yet?